SCons/Tool/ninja/ninja.xml

```diff
@@ -350,15 +350,15 @@ python -m pip install ninja
         <summary>
             <para>
                 Internal value used to specify the function to call with argument env to generate the list of files
-                which if changed would require the &ninja; build file to be regenerated.
+                which -- if changed -- would require the &ninja; build file to be regenerated.
```

Comment:
not super fond of double dashes as separators, though that may be only a personal bias.

Reply:
Can also use parentheses instead if that gets this PR going.

Reply:
proper grammar would be `which, if changed, would...`